build-support/bin/packages.py

```diff
@@ -417,6 +444,73 @@ def build_fs_util() -> None:
         green(f"Built fs_util at {dest_dir / 'fs_util'}.")
 
 
+# TODO: We should be using `./pants package` and `pex_binary` for this...If Pants is lacking in
+#  capabilities, we should improve Pants. When porting, using `runtime_package_dependencies` to do
+#  the validation.
+def build_pex(fetch: bool) -> None:
+    linux_platform = "linux_x86_64"
+    macos_platform = "macosx_10.15_x86_64"
+    if fetch:
+        extra_pex_args = [
+            f"--platform={plat}-{abi}"
+            for plat in (linux_platform, macos_platform)
+            for abi in ("cp-37-m", "cp-38-cp38", "cp-39-cp39")
+        ]
+        pex_name = f"pants.{CONSTANTS.pants_unstable_version}.pex"
+        banner(f"Building {pex_name} by fetching wheels.")
+    else:
+        extra_pex_args = [f"--python={sys.executable}"]
+        plat = os.uname()[0].lower()
+        py = f"cp{''.join(map(str, sys.version_info[:2]))}"
+        pex_name = f"pants.{CONSTANTS.pants_unstable_version}.{plat}-{py}.pex"
```

Comment:
This naming scheme is tweaked. Before, `{platform}` was undefined in Bash due to a bad change earlier, and `{py}` was not specified.